<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meterAddress</class>
 <widget class="QDialog" name="meterAddress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix/img/device_mm_router_wired_128px_42870_easyicon.net.ico</normaloff>:/new/prefix/img/device_mm_router_wired_128px_42870_easyicon.net.ico</iconset>
  </property>
  <widget class="QPushButton" name="pushBtnOk">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnTest">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>万用表IP：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>169.254.1.61</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5025</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatusIcon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/new/prefix/img/start.bmp</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatusInfo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>万用表未连接</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
